internal.h

```diff
@@ -86,13 +98,18 @@ void fcgi_stop (void);
 /* Rest */
 extern bool rest_use_arrays;
 extern bool rest_use_types;
-gboolean rest_init (const char *path);
+gboolean rest_init (const char *path, const char *logging);
 void rest_api (req_handle handle, int flags, const char *rpath, const char *path,
                const char *if_match, const char *if_none_match,
                const char *if_modified_since, const char *if_unmodified_since,
-               const char *server_name, const char *server_port, const char *data, int length);
+               const char *server_name, const char *server_port,
+               const char *remote_addr, const char *remote_user,
+               const char *data, int length);
 void rest_shutdown (void);
 void yang_library_create (sch_instance *schema);
 void restconf_monitoring_create (sch_instance *schema);
+bool restconf_logging_test_flag (int flag);
```

Comment:
these do not need a "restconf" prefix ... just logging_init, logging_shutdown, logging_test_flag

Reply:
Done